iset.mm

```diff
@@ -145042,6 +145058,229 @@ mathematician Ptolemy (Claudius Ptolemaeus).  This particular version is
+     negative.  (Contributed by NM, 17-Aug-2008.) $)
+  sinq34lt0t $p |- ( A e. ( _pi (,) ( 2 x. _pi ) ) -> ( sin ` A ) < 0 ) $=
+    ( cpi c2 cmul co cioo wcel csin cfv cc0 clt cneg cmin cr wb caddc pire picn
+    wbr syl elioore wa 0re iooshf mpanr12 addid2i eqcomi 2timesi oveq12i eleq2i
+    mpan2 syl6rbbr ibi sinq12gt0 cc wceq recnd sinmpi breqtrd resincld lt0neg1d
+    mpbird ) ABCBDEZFEZGZAHIZJKSJVFLZKSVEJABMEZHIZVGKVEVHJBFEGZJVIKSVEVJVEANGZV
+    EVJOABVCUAZVKVJAJBPEZBBPEZFEZGZVEVKBNGZVJVPOZQVKVQUBJNGVQVRUCQABJBUDUEUKVDV
+    OABVMVCVNFVMBBRUFUGBRUHUIUJULTUMVHUNTVEAUOGVIVGUPVEAVLUQAURTUSVEVFVEAVLUTVA
+    VB $.
+
+  $( The cosine of a number strictly between ` -u _pi / 2 ` and ` _pi / 2 ` is
+     positive.  (Contributed by Mario Carneiro, 25-Feb-2015.) $)
+  cosq14gt0 $p |- ( A e. ( -u ( _pi / 2 ) (,) ( _pi / 2 ) ) ->
+                    0 < ( cos ` A ) ) $=
+    ( cpi co cioo wcel cc0 cmin cfv clt wbr cr halfpire cxr w3a wb rexri elioo2
+    cc picn syl cdiv cneg csin ccos elioore resubcl sylancr neghalfpirx simp3bi
+    mp2an posdif sylancl mpbid halfcl ax-mp negcli caddc pidiv2halves subaddrii
+    c2 negsubi eqtri simp2bi eqbrtrid pire ltsub23 mp3an13 mpbird 0xr syl3anbrc
+    sinq12gt0 wceq recnd sinhalfpim breqtrd ) ABUTUACZUBZVPDCEZFVPAGCZUCHZAUDHZ
+    IVRVSFBDCEZFVTIJVRVSKEZFVSIJZVSBIJZWBVRVPKEZAKEZWCLAVQVPUEZVPAUFUGVRAVPIJZW
+    DVRWGVQAIJZWIVQMEVPMEVRWGWJWINOUHVPLPVQVPAQUJZUIVRWGWFWIWDOWHLAVPUKULUMVRWE
+    VPBGCZAIJZVRWLVQAIVPBVQBREVPRESBUNUOZSVPWNUPBVQUQCBVPGCVPBVPSWNVABVPVPSWNWN
+    URUSVBUSVRWGWJWIWKVCVDVRWGWEWMOZWHWFWGBKEWOLVEVPABVFVGTVHFMEBMEWBWCWDWENOVI
+    BVEPFBVSQUJVJVSVKTVRAREVTWAVLVRAWHVMAVNTVO $.
+
+  $( The cosine of a number in the second and third quadrants is negative.
+     (Contributed by Jim Kingdon, 14-Mar-2024.) $)
+  cosq23lt0 $p |- ( A e. ( ( _pi / 2 ) (,) ( 3 x. ( _pi / 2 ) ) ) ->
+      ( cos ` A ) < 0 ) $=
+    ( cpi c2 cdiv co c3 cmul wcel caddc cc0 clt cc wbr cr halfpire cxr rexri c4
+    c1 oveq1i cioo csin cfv ccos wceq elioore recnd sinhalfpip syl a1i readdcld
+    pidiv2halves w3a wb remulcli elioo2 mp2an simp2bi ltadd2dd eqbrtrrid ax-1cn
+    3re simp3bi 3cn recni adddiri addcomli mulid2i 3eqtr3ri cap wa 4cn 2cn 2ap0
+    3p1e4 pm3.2i picn div32ap mp3an 4d2e2 3eqtr2i breqtrdi syl3anbrc sinq34lt0t
+    pire 2re eqbrtrrd ) ABCDEZFWHGEZUAEHZWHAIEZUBUCZAUDUCZJKWJALHWLWMUEWJAAWHWI
+    UFZUGAUHUIWJWKBCBGEZUAEHZWLJKMWJWKNHZBWKKMZWKWOKMZWPWJWHAWHNHWJOUJZWNUKWJBW
+    HWHIEWKKULWJWHAWHWTWNWTWJANHZWHAKMZAWIKMZWHPHWIPHWJXAXBXCUMUNWHOQWIFWHVBOUO
+    ZQWHWIAUPUQZURUSUTWJWKWHWIIEZWOKWJAWIWHWNWINHWJXDUJWTWJXAXBXCXEVCUSXFRWHGEZ
+    RCDEZBGEZWOSFIEZWHGESWHGEZWIIEXGXFSFWHVAVDWHOVEZVFXJRWHGFSRVDVAVOVGTXKWHWII
+    WHXLVHTVIRLHCLHZCJVJMZVKBLHXIXGUEVLXMXNVMVNVPVQRCBVRVSXHCBGVTTWAWBBPHWOPHWP
+    WQWRWSUMUNBWEQWOCBWFWEUOQBWOWKUPUQWCWKWDUIWG $.
+
+  $( Location of the zeroes of cosine in
+     ` ( -u ( _pi / 2 ) (,) ( 3 x. ( _pi / 2 ) ) ) ` .  (Contributed by Jim
+     Kingdon, 14-Mar-2023.) $)
```

Comment:
Should this be 14-Mar-2024, shoudn't it?

Reply:
Good eyes. You are correct.

Fixed.